artifacts/centos/centos_test.go

```diff
@@ -29,6 +29,7 @@ var _ = Describe("Centos", func() {
 				DownloadURL:          "https://cloud.centos.org/centos/8/x86_64/images/CentOS-8-GenericCloud-8.4.2105-20210603.0.x86_64.qcow2",
 				Compression:          "",
 				AdditionalUniqueTags: []string{"8.4.2105-20210603.0", "8.4.2105"},
+				ImageArchitecture:    "amd64",
```

Comment:
I would export this as variable

Reply:
It could be a const, yes.

Reply:
Thanks, I'll take care of it in another PR where we'll add also `arm64` images.